pkg/rancher-desktop/components/MarketplaceCatalog.vue

```diff
@@ -67,6 +59,20 @@ export default (Vue as VueConstructor<Vue & VuexBindings>).extend({
     filteredExtensions(): FilteredExtensions {
       let tempExtensions = this.extensions;
 
+      tempExtensions = tempExtensions.map((item) => {
```

Comment:
We should be able to distill this down to

```suggestion
      let tempExtensions = this.extensions.map((item) => {
```